src/content/docs/en/pages/devtools/cli/azion-cli-alpha/edge-functions/edge-functions.mdx

```diff
@@ -1,12 +1,16 @@
 ---
-title: Edge function
+title: Azion CLI edge-function
 description: Create, list, describe, update, and delete your edge functions on the Azion platform through Azion CLI.
 meta_tags: cli, edge functions, devops
 namespace: documentation_cli_edge_functions
 menu_namespace: cliMenuAlpha
 permalink: /documentation/products/cli/edge-functions-crud/
 ---
 
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case any required field isn't informed through the specific flags, an interactive message prompts and asks for the missing information.
```